### DISCUSSION TITLE
monthly bills

### DISCUSSION THREAD

**Discussion by shahram7** [spiral-project/ihatemoney#1320] [Q&A]
Hi everyone,
Is it possible to create a monthly bill? I am a little bit confused because sometimes I see this option in MoneyBuster, and other times I am not able to find it again.
Best regards, Shahram

**Comment by almet** [comment]
Hi, it is not possible at the moment on ihatemoney. It is possible via Money Buster if you're using a cospend backend, though.

There is an issue about this here: #152
